contrib/builder/rpm/ppc64le/build.sh

```diff
@@ -3,6 +3,8 @@ set -e
 
 cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
+wget http://download.opensuse.org/repositories/Virtualization:/containers:/images:/openSUSE-Tumbleweed/images/openSUSE-Tumbleweed-docker-guest-docker.ppc64le.tar.xz && unxz openSUSE-Tumbleweed-docker-guest-docker.ppc64le.tar.xz && docker load -i openSUSE-Tumbleweed-docker-guest-docker.ppc64le.tar 
```

Comment:
@clnperez we don't need this right? This was just for testing tumbleweed when we didn't have an image for it?